Tools/clinic/clinic.py

```diff
@@ -2149,6 +2156,13 @@ def print_block(
 
             """)
 
+            if clinic:
+                # Emit optional includes
+                for include, reason in sorted(clinic.includes.items()):
+                    line = f'#include "{include}"'
+                    line = line.ljust(35) + f'// {reason}\n'
```

Comment:
Should we add a comment to explain the indent amount?
```suggestion
                    include = f'"{include}"'
                    line = f'#include {include: <25} // {reason}\n'
```

Reply:
35 is my arbitrary convention that I'm using in Python. I prefer to keep ljust(35) since it reminds me the 35th column :-)